### DISCUSSION TITLE
请问官网的主题模板是什么？

### DISCUSSION THREAD

**Discussion by googidaddy** [clash-verge-rev/clash-verge-rev#2039] [General]
https://www.clashverge.dev/

一直好奇。。。官网主题模板是啥

**Comment by Dragon1573** [comment]
主题名字叫做 Material ，整个站点使用 mkdocs 框架。

配置文件可以在 [这里](https://github.com/clash-verge-rev/clash-verge-rev.github.io/blob/2c4f1b3fa613624fa1e4758ee024a28e4fc23f68/mkdocs.yml#L48-L81) 找到。
